package.json

```diff
@@ -84,9 +84,9 @@
     "airbnb-prop-types": "^2.15.0",
     "object.assign": "^4.1.0",
     "prop-types": "^15.7.2",
-    "react-with-direction": "^1.3.0",
-    "react-with-styles": "^3.2.3",
-    "react-with-styles-interface-css": "^4.0.3"
+    "react-with-direction": "^1.3.1",
+    "react-with-styles": "^4.0.0",
+    "react-with-styles-interface-css": "^5.0.0"
```

Comment:
- `react-with-direction@1.3.1` is required to work with `react-with-styles@4.0.0`
- `react-with-styles-interface-css@5.0.0` has `react-with-styles@4.0.0` as a peer dep